win32ss/user/user32/windows/class.c

```diff
@@ -1841,7 +1841,7 @@ SetWindowLongPtrA(HWND hWnd,
                   INT nIndex,
                   LONG_PTR dwNewLong)
 {
-    return NtUserSetWindowLong(hWnd, nIndex, dwNewLong, FALSE);
+    return NtUserSetWindowLongPtr(hWnd, nIndex, dwNewLong, FALSE);
```

Comment:
The last boolean should be TRUE here (calling the function in ANSI mode).
The other call in SetWindowLongPtrW is OK however.